src/coreclr/vm/corhost.cpp

```diff
@@ -770,7 +770,7 @@ HRESULT CorHost2::Authenticate(ULONGLONG authKey)
     CONTRACTL
     {
         NOTHROW;
-        if (GetThreadNULLOk()) {GC_TRIGGERS;} else {DISABLED(GC_NOTRIGGER);}
+        if (GetThreadNULLOk()) {DISABLED(GC_NOTRIGGER);} else {GC_TRIGGERS;}
```

Comment:
```suggestion
        GC_TRIGGERS;
```
Or it can be LIMITED_METHOD_CONTRACT;. This does not do anything...